src/core/dxf/qgsdxfexport.h

```diff
@@ -125,6 +132,11 @@ class CORE_EXPORT QgsDxfExport : public QgsLabelSink
          * \brief Limit for the number of data defined symbol block classes (keep only the most used ones). -1 means no limit
          */
         int mDDBlocksMaxNumberOfClasses = -1;
+
+        /**
+         * \brief Overridden name of the layer to be exported to DXF
+         */
+        QString mOverriddenName = QString();
```

Comment:
```suggestion
        QString mOverriddenName;
```

Reply:
Addressed in https://github.com/qgis/QGIS/pull/57016/commits/fe2a49cad1405e924fa9484c1bd6f0012945a0c8m thanks!